source/community/reactnative/src/components/inputs/choice-set/choice-set-input.js

```diff
@@ -42,10 +42,10 @@ export class ChoiceSetInput extends React.Component {
 		this.payload = props.json;
 		this.label = Constants.EmptyString;
 		this.isRequired = this.payload.isRequired || false;
+		this.placeholder = props.json.placeholder
```

Comment:
as props.json is assigned to this.payload, it is better to use this.payload in all places instead of props.json